modules/local/check_torch_model.nf

```diff
@@ -7,18 +7,21 @@ process CHECK_TORCH_MODEL {
     
     input:
     tuple path(original_csv), path(model),  path(experiment_config), path(ray_tune_config)
+    each path(initial_weights)
 
     output:
     stdout emit: standardout
 
     script:
     def args = task.ext.args ?: ''
+    def args_initial_weights = initial_weights ? "--initial_weights $initial_weights" : ""
     """
     launch_check_model.py \
         -d ${original_csv} \
         -m ${model} \
         -e ${experiment_config} \
         -c ${ray_tune_config} \
+        ${args_initial_weights} \
```

Comment:
```suggestion
        --initial_weights ${args_initial_weights} \
```

Reply:
why this work? instead of defining args_initial_weights?